packages/protect/src/lib/protect.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+ * Copyright © 2025 Ping Identity Corporation. All right reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ *
+ */
+
+import { ProtectConfig, Protect } from './protect.types.js';
+
+// Add Signals SDK namespace to the window object
+declare global {
+  interface Window {
+    _pingOneSignals: {
+      init: (initParams?: ProtectConfig) => Promise<void>;
+      getData: () => Promise<string>;
+      pauseBehavioralData: () => void;
+      resumeBehavioralData: () => void;
+    };
+  }
+}
+
+/**
+ * @function createProtect - returns a set of methods to interact with the PingOne Signals SDK
+ * @param {ProtectConfig} config - the configuration options for the PingOne Signals SDK
+ * @returns {Protect} - a set of methods
+ */
+export function createProtect(config: ProtectConfig): Protect {
+  const options = config;
+  return {
+    start: async function start(): Promise<void> {
+      try {
+        /*
+         * Load the Ping Signals SDK
+         * this automatically pollutes the window
+         * there are no exports of this module
+         */
+        await import('./signals-sdk.js' as string);
+      } catch (err) {
+        console.error('error loading ping signals', err); // TODO: should we throw an error here?
```

Comment:
@cerebrl @ryanbas21 I copied this from the legacy sdk but left one comment here. Wondering if we should throw an error if the signals sdk fails to load. Currently if it fails then it will continue to try to initialize anyways... but that will also fail because no methods would have been added to `window._pingOneSignals`

Reply:
I think throwing an error here makes sense. 

@cerebrl I'm curious if maybe it makes more sense to load the signals-sdk in the closure? That way if the signals SDK fails to load, we don't actually return the returning object so there's no confusion on the api "working" at that level.

Obviously this would make the factory async, which is a bit different than what we've typically done. But I think it could be useful to error (if we do) at the creation stage, because without the signals-sdk the api is useless.

Reply:
Yeah, I think that's a good idea.